app/services/Worker.php

```diff
@@ -3,70 +3,83 @@
 namespace GanttDashboard\App\Services;
 
 use GanttDashboard\App\Models\Workers;
-use Phalcon\Session\Adapter\Files as SessionAdapter;
-use Phalcon\Security;
+use GanttDashboard\App\Validators\Worker as WorkerValidator;
+use GanttDashboard\App\Services\Authentication as AuthenticationService;
```

Comment:
is this used here?

Reply:
Yes, but for my next commit I'll remove the validator from here. The authentication is used for hashing the hashPassword method in case the new worker is admin.